datadog-opentelemetry/Cargo.toml

```diff
@@ -43,7 +43,7 @@ datadog-opentelemetry = { path = ".", features = ["test-utils"] }
 
 # Testing the tracing API
 tracing = { version = "0.1.4" }
-tracing-opentelemetry = { version = "0.31" }
+tracing-opentelemetry = { version = "0.32" }
```

Comment:
Shouldn't this be `0.32.0`?

Reply:
The way I read that, yes, since otherwise we would only get `0.31.1` an not `0.31.2`. But who knows 🤷🏼‍♂️